### PR TITLE
Mission comleted! Catalog was rewritten in grid

### PR DESCRIPTION
**Pull request by RatuhWD**
- [DEMO LINK](https://ratyhwd.github.io/layout_catalog_grid/)

### CODE REVIEW

**Comment by RatuhWD** [comment]
- [DEMO LINK](https://ratuhwd.github.io/layout_catalog_grid/)

File: src/styles/main.scss [hunk 1]
```diff
@@ -12,3 +12,24 @@ body {
 
 @import "_header.scss";
 @import "_catalog.scss";
+
+.catalog {
+  display: grid;
+
+  justify-content: center;
+  grid-template-columns: 200px 200px 200px 200px;
```
Comment (sTorba24) [inline src/styles/main.scss:20]: use `repeat( number, size)` for your case it is `repeat(4, 200px)`
the same in media queries :)

File: src/styles/main.scss [hunk 2]
```diff
@@ -12,3 +12,24 @@ body {
...
+  grid-template-rows: repeat(auto-fill, 300px);
```
Comment (sTorba24) [inline src/styles/main.scss:21]: you can remove i think

**Review by sTorba24 — CHANGES_REQUESTED** [review]
Nice job! some comments to improve :)

**Pull request closed without merging**